_config/versioned.yml

```
@@ -0,0 +1,21 @@
+---
+Name: graphql-versioned-schema
+Only:
+  moduleexists: 'silverstripe/versioned'
+---
+SilverStripe\GraphQL\Schema\Schema:
+  schemas:
+    '*':
+      src:
+        versionedSrc: 'silverstripe/graphql: _config/_versioned'
```

```suggestion
        versionedSrc: 'silverstripe/graphql: _config/versioned'
```
I'm also not sure what that `silverstripe/graphql` bit does - if it still works when you remove that, you should probably remove it.